app/services/cms/providers/strapi/factories/component_factory.rb

```diff
@@ -27,6 +27,26 @@ def self.process_component(strapi_data)
                 colour_theme: strapi_data.dig(:colourTheme, :data) ? strapi_data[:colourTheme][:data][:attributes][:name] : nil,
                 icon_block: icon_block(strapi_data[:iconBlock])
               )
+            when "blocks.card-section"
+              DynamicComponents::CardWrapper.new(
+                title: strapi_data[:sectionTitle],
+                cards_block: resource_card_block(strapi_data[:resourceCard]),
+                cards_per_row: strapi_data[:cardsPerRow],
+                background_color: strapi_data.dig(:backgroundColour, :data) ? strapi_data[:backgroundColour][:data][:attributes][:name] : nil
+              )
+            end
+          end
+
+          def self.resource_card_block(strapi_data)
+            strapi_data.map do |card_data|
+              Models::ResourceCard.new(
+                title: card_data[:title],
+                icon: card_data.dig(:icon, :data) ? ModelFactory.to_image(card_data[:icon][:data][:attributes]) : nil,
+                colour_theme: card_data.dig(:colourTheme, :data) ? card_data[:colourTheme][:data][:attributes][:name] : nil,
+                body_text: card_data[:bodyText],
```

Comment:
It might be worth wrapping these bits that use the Rich Block from strapi in `ModelFactory.to_content_block` as this auto processes any images they insert into the content block. It might not be a case that they are going to put images in, but without it, it will crash.